test-utils/src/main/kotlin/org/cqfn/save/testutils/MockWebServerUtils.kt

```diff
@@ -0,0 +1,137 @@
+@file:Suppress("FILE_NAME_MATCH_CLASS", "MatchingDeclarationName")
+
+package org.cqfn.save.testutils
+
+import okhttp3.mockwebserver.*
+import org.junit.Assert.assertTrue
+import org.slf4j.Logger
+import java.net.HttpURLConnection
+import java.nio.charset.Charset
+import java.util.concurrent.*
+
+private typealias ResponsesMap = ConcurrentMap<String, BlockingQueue<MockResponse>>
+
+/**
+ * Queue dispatcher with additional logging
+ *
+ * @param logger
+ */
+class LoggingQueueDispatcher(private val logger: Logger) : Dispatcher() {
+    private val responses: ResponsesMap = ConcurrentHashMap()
+    private val defaultResponses: ConcurrentMap<String, MockResponse> = ConcurrentHashMap()
+    private var failFastResponse: MockResponse = MockResponse().setResponseCode(HttpURLConnection.HTTP_NOT_FOUND)
+
+    private fun getMethodPath(fullPath: String?) = fullPath?.let { Regex("/[\\w]*").find(it)?.value } ?: ""
+
+    @Suppress("UnsafeCallOnNullableType", "AVOID_NULL_CHECKS")
+    override fun dispatch(request: RecordedRequest): MockResponse {
+        val path = getMethodPath(request.path)
+        val result = if (defaultResponses[path] != null) {
+            logger.info("Default response [${defaultResponses[path]}] exists for path [$path].")
```

Comment:
```suggestion
            logger.debug("Default response [${defaultResponses[path]}] exists for path [$path].")
```
Also, I think it would make sense to create a separate logger for this class instead of providing one and then tweak its level via `resources/test/application.properties`

Reply:
Ok, done